torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -59,6 +57,11 @@ def __call__(
         return super().__call__(subgraph, identifier, operands)
 
 
+class InvokeSubgraphHOP(InvokeSubgraphHOPImpl):
+    def __init__(self) -> None:
+        super().__init__("invoke_subgraph")
```

Comment:
What are the semantics of `invoke_quant(graph, args, scheme)` ? Can we add a docstring that says what it does?